app/models/delivery_partner_profile.rb

```diff
@@ -6,6 +6,8 @@ class DeliveryPartnerProfile < ApplicationRecord
   belongs_to :user
   belongs_to :delivery_partner
 
+  scope :with_users, -> { includes(:user) }
```

Comment:
I wonder if we do need this scope, as we're using it only once 🤔

Reply:
@cwrw 
sure, I'll get rid of it